pkg/host/host.go

```diff
@@ -47,6 +47,7 @@ const (
 	FeatureComparisons
 	FeatureSandboxSetuid
 	FeatureSandboxNamespace
+	FeatureSandboxAndroidUntrustedApp
```

Comment:
Also add it in `func Check` below. It needs a human-readable name (that fits into 24 chars because we print them with `%-24v` in aligned table output).